app/src/ui/octicons/octicons.generated.ts

```diff
@@ -195,7 +195,14 @@ export const calendar: OcticonSymbolType = {
 export const check: OcticonSymbolType = {
   w: 16,
   h: 16,
-  d: 'M13.78 4.22a.75.75 0 010 1.06l-7.25 7.25a.75.75 0 01-1.06 0L2.22 9.28a.75.75 0 011.06-1.06L6 10.94l6.72-6.72a.75.75 0 011.06 0z',
+  d: 'M13.78 4.22a.75.75 0 0 1 0 1.06l-7.25 7.25a.75.75 0 0 1-1.06 0L2.22 9.28a.751.751 0 0 1 .018-1.042.751.751 0 0 1 1.042-.018L6 10.94l6.72-6.72a.75.75 0 0 1 1.06 0Z',
+  fr: 'evenodd',
+}
+
+export const diffCheck: OcticonSymbolType = {
+  w: 12,
+  h: 12,
+  d: 'M10.5303 2.96967C10.8232 3.26256 10.8232 3.73744 10.5303 4.03033L5.03033 9.53033C4.73744 9.82322 4.26256 9.82322 3.96967 9.53033L1.46967 7.03033C1.17678 6.73744 1.17678 6.26256 1.46967 5.96967C1.76256 5.67678 2.23744 5.67678 2.53033 5.96967L4.5 7.93934L9.46967 2.96967C9.76256 2.67678 10.2374 2.67678 10.5303 2.96967Z',
```

Comment:
Just curious, where did these changes come from? I'd expect it to come from some other change (like updating the `@primer/octicons` package) since [this file is supposed to be auto-generated](https://github.com/desktop/desktop/blob/d3f7c5ec7e002b27c7d84638164a59cfb8a34c74/app/src/ui/octicons/octicons.generated.ts#L1-L4).

Reply:
For icons that don't come from `@primer/octicons`, I see we declare them in specific files where needed, for example: https://github.com/desktop/desktop/blob/d3f7c5ec7e002b27c7d84638164a59cfb8a34c74/app/src/ui/octicons/sync-clockwise.ts/#L7

Reply:
Ah ok . I didn't know that convention. I will make iit it's own file. Gavin made the new checkmark because the primer one was very thin especially when reduced to 12px. I will move this. ... 🤦 I used search to find the octicon file and didn't even notice the title was "generated".

Reply:
done